include/deviceconfig.h

```diff
@@ -94,7 +94,9 @@
 
 #define DEVICE_CONFIG_FILE          "/device.cfg"
 #define NTP_SERVER_DEFAULT          "0.pool.ntp.org"
-#define BRIGHTNESS_MIN              uint8_t(10)
+#ifndef BRIGHTNESS_MIN
+    #define BRIGHTNESS_MIN          uint8_t(10)
+#endif
 #define BRIGHTNESS_MAX              uint8_t(255)

```

Comment:
Allows non-matrix effects to set the brightness to 0 if the need to, like lamp for example